src/proformtreadmill.cpp

```diff
@@ -819,7 +825,7 @@ void proformtreadmill::update() {
             if (counterPoll > 5) {
                 counterPoll = 0;
             }
-        } else if (nordictrack_s30_treadmill) {
+        } else if (nordictrack_s30_treadmill || nordictrack_z1300i_treadmill) {
```

Comment:
Not sure about what data is needed. Model S30 keeps connection alive and displays data from trainer. Data could be the same. Could you check bluetooth logs to verify this?

Reply:
Ok let's try first the build, then I will check the log tomorrow. Let me know